tests/functional/ContentCest.php

```diff
@@ -0,0 +1,304 @@
+    }
+
+
+    public function canViewCategory(FunctionalTester $I)
+    {
+        $category = $I->haveContent(['type' => 'category', 'isActive' => 1]);
+        $content  = $I->haveContent(['type' => 'content', 'isActive' => 1, 'parentId' => $category->id]);
+        $route    = '/' . $category->route->translations[0]['langCode'] . '/' . $category->route->translations[0]['url'];
+
+        $I->wantTo('view category');
+        $I->amOnPage($route);
+        $I->seeResponseCodeIs(200);
+
+        $I->see($category->translations[0]->title);
+        $I->see($content->translations[0]->title);
+    }
+
+
+    public function canGoToArticleFromCategory(FunctionalTester $I)
+    {
+        $category      = $I->haveContent(['type' => 'category', 'isActive' => 1]);
+        $content       = $I->haveContent(['type' => 'content', 'isActive' => 1, 'parentId' => $category->id]);
+        $contentRoute  = '/' . $content->route->translations[0]['langCode'] . '/' . $content->route->translations[0]['url'];
+        $categoryRoute = '/' . $category->route->translations[0]['langCode'] . '/' . $category->route->translations[0]['url'];
+
+        $I->wantTo('read more');
+        $I->amOnPage($categoryRoute);
+        $I->seeResponseCodeIs(200);
+
+        $I->click('Read more');
+        $I->canSeeCurrentUrlEquals($contentRoute);
+    }
+
+    public function canSeeNotPublishedContentAsAdmin(FunctionalTester $I)
+    {
+        $category     = $I->haveContent(['type' => 'category', 'isActive' => 1]);
+        $content      = $I->haveContent(['type' => 'content', 'isActive' => 0, 'parentId' => $category->id]);
+        $contentRoute = '/' . $content->route->translations[0]['langCode'] . '/' . $content->route->translations[0]['url'];
+
+        $I->wantTo('see not published content as admin user');
+        $I->loginAsAdmin();
+        $I->amOnPage($contentRoute);
+        $I->seeResponseCodeIs(200);
+
+        $I->see($content->translations[0]->title);
+        $I->see('This content is not published.');
+    }
+
+    public function cantSeeNotPublishedContentAsRegularUser(FunctionalTester $I)
+    {
+        $content = $I->haveContent(['type' => 'content', 'isActive' => 0]);
+        $route   = '/' . $content->route->translations[0]['langCode'] . '/' . $content->route->translations[0]['url'];
+
+        $I->wantTo('try to see not published content as regular user');
+        $I->amOnPage($route);
+        $I->seeResponseCodeIs(404);
+    }
+
+    public function seeStickyContentOnTopOfTheList(FunctionalTester $I)
+    {
+        $category         = $I->haveContent(['type' => 'category', 'isActive' => 1]);
+        $stickyContent    = $I->haveContent(
+            [
+                'type'         => 'content',
+                'isActive'     => 1,
+                'isSticky'     => 1,
+                'parentId'     => $category->id,
+                'translations' => [
+                    'langCode' => 'en',
+                    'title'    => 'ZZZ',
```

Comment:
I think that we should use title with meaning, rather than some random letters, eg. "Sticky Content"